### DISCUSSION TITLE
Flashing using Raspberry pi 4 gpio as SWD protocol

### DISCUSSION THREAD

**Discussion by diazmanuel** [pyocd/pyOCD#1413] [Q&A]
Hello could you tell me if its possible to use the gpio of the raspberry pi 4 to simulate a swd protocol
i need to program a microchip of nrf
thanks for the time and sorry if its some off topic issue

**Comment by flit** [comment]
Hi @diazmanuel. pyOCD doesn't have a probe driver for Raspberry Pi GPIO. It's been asked about, and someone seemed interested in writing a driver, but nothing has happened yet. This would be a great first contribution for someone.
